dbsa/presto.py

```diff
@@ -166,3 +166,33 @@ def get_create_current_partition_view(self, suffix='_latest', condition='', igno
             select=self.get_select_current_partition(condition=condition, ignored_partitions=ignored_partitions, params=params, transforms=transforms),
             suffix=suffix,
         )
+
+    def get_incremental_update_select(self, update_select, row_identifier=None, filter_fn=None, condition='', ignored_partitions=None, params=None, transforms=None):
+        return Template("""
+            WITH incremental_update AS (
+                {{ update_select }}
+            ),
+            recent_data AS (
+                {{ select }}
+            )
+            SELECT *
```

Comment:
Remove `*`, and list the table columns here to ensure both query has the same columns and column order.

Reply:
The ctes already defines the column order. The query that you pass has to match the order of the managed current partition select which by default excludes partitions, so you only need to match that. if we require a list of fields, that needs to be specified in all cases, so usage of the function would be more complex.